{{cookiecutter.project_slug}}/backend/make-requirements.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+poetry export -f requirements.txt -o requirements.txt $(if [[ "$1" == "--dev" ]]; then echo "--dev"; fi)
```

Comment:
I would prefer not to have this file that requires Poetry installed locally to work and instead add poetry to the Dockerfiles.